src/lib/forms/core/RJSWidgets.js

```diff
@@ -0,0 +1,178 @@
+import { withCancel } from '@api/utils';
```

Comment:
for the filename, maybe it would be better to make them specialised f.e. RJSVocabularyWidgets.js? I think the widgets will grow in the future